### PR TITLE
Consistent format for SDS object descriptions

### PR DESCRIPTION
**Pull request by mckenzielee**
Used consistent format and spacing for all SDS objects.

### CODE REVIEW

File: Documentation/SequentialDataStore/Units_of_Measure.md [hunk 1]
```diff
@@ -301,241 +299,240 @@ Returns the quantity corresponding to the specified quantityId within a given na
...
-                0,
-                0,
-                0,
-                0
-            ]
-        }
+Example response body for quantityId = "Length":
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Units_of_Measure.md]: Line 320 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/Units_of_Measure.md [hunk 2]
```diff
@@ -614,46 +609,45 @@ The response includes a status code and a response body.
...
-                "QuantityId": "Electric Charge",
-                "ConversionFactor": 1
-            }
-            ...
-        ]
-
+Example response body:
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Units_of_Measure.md]: Line 615 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/Units_of_Measure.md [hunk 3]
```diff
@@ -548,41 +545,39 @@ Returns the unit of measure corresponding to the specified uomId within a given
...
-            "Name": "ounce",
-            "DisplayName": "ounce",
-            "QuantityId": "Mass",
-            "ConversionFactor": 0.028349523
-        }
-
+Example response body for uomId = "ounce":
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Units_of_Measure.md]: Line 566 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/Units_of_Measure.md [hunk 4]
```diff
@@ -301,241 +299,240 @@ Returns the quantity corresponding to the specified quantityId within a given na
...
-                "Name": "ampere",
-                "DisplayName": "ampere",
-                "QuantityId": "Electric Current",
-                "ConversionFactor": 1
-            }
-        ]
+Example response for quantityId = "Electric Current":
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Units_of_Measure.md]: Line 507 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/Units_of_Measure.md [hunk 5]
```diff
@@ -301,241 +299,240 @@ Returns the quantity corresponding to the specified quantityId within a given na
...
-                "Name": "mile",
-                "DisplayName": "mile",
-                "QuantityId": "Length",
-                "ConversionFactor": 1609.344
-        }
-
+Example response for quantityId = "Length" and uomId ="mile":
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Units_of_Measure.md]: Line 464 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/Units_of_Measure.md [hunk 6]
```diff
@@ -301,241 +299,240 @@ Returns the quantity corresponding to the specified quantityId within a given na
...
-                    0,
-                    0,
-                    0
-                ]
-            },
-            ...
+Example response body:
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/Units_of_Measure.md]: Line 381 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 7]
```diff
@@ -624,96 +594,88 @@ in the Namespace. See [Searching](xref:sdsSearching) and [Filter Expressions: Me
...
-               },
-               {
-                  "SourceId":"Measurement",
-                  "TargetId":"Value"
-               }				 
-            ]
-         }
-       ]
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Views.md]: Line 636 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 8]
```diff
@@ -546,70 +525,61 @@ Returns the stream view map corresponding to the specified streamViewId within a
...
+The requested SdsStreamView.
 
-  Sample response body:
-
-      HTTP/1.1 200
-      Content-Type: application/json
+Example response body:
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Views.md]: Line 547 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Views.md [hunk 9]
```diff
@@ -104,440 +100,423 @@ The available SdsStreamViewModes are shown in the table below.
...
-
-      HTTP/1.1 200
-      Content-Type: application/json
+**Response body**  
+The requested SdsStreamView.
 
+Example response body:
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Views.md]: Line 485 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Types.md [hunk 10]
```diff
@@ -800,25 +787,23 @@ Returns the type corresponding to the specified typeId within a given namespace.
...
-
-  The requested SdsType
+**Response body**  
+The requested SdsType
 
-  Sample response body:
+Example response body:
 ```json
 HTTP/1.1 200
 Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Types.md]: Line 809 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Types.md [hunk 11]
```diff
@@ -898,40 +882,38 @@ for information about specifying those respective parameters.
...
-**Response body**
-
+**Response body**  
 A collection of zero or more SdsTypes
 
-Sample response body:
+Example response body:
 ```json
 HTTP/1.1 200
 Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Types.md]: Line 919 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Types.md [hunk 12]
```diff
@@ -1088,17 +1065,20 @@ Sample SdsType content:
...
 The response includes a status code and a response body.
 
-Response body
+**Response body**  
+The request content is the serialized SdsType. If you are not using the SDS Client Libraries, it is recommended that you use JSON.
+
+Example Response body:
 ```json
 HTTP/1.1 201
 Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Types.md]: Line 1078 should be: Content-Type: application/json
Line 1080 should not be in a response message

File: Documentation/SequentialDataStore/SDS_Streams.md [hunk 13]
```diff
@@ -109,64 +101,51 @@ SdsStream information.
...
 
-      {  
-         "Id":"Simple"
-         "Name":"Simple"
-         "TypeId":"Simple",
-      }
+Example response body:
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Streams.md]: Line 132 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Streams.md [hunk 14]
```diff
@@ -179,132 +158,111 @@ for information about specifying those respective parameters.
...
+**Response**  
+The response includes a status code and a response body.
+
+**Response body**  
+A collection of zero or more SdsStreams.
+
+Example response body:
+```json
+HTTP/1.1 200
+Accept: application/json
```
Comment (ScottPatterson) [inline Documentation/SequentialDataStore/SDS_Streams.md]: Line 201 should be: Content-Type: application/json

File: Documentation/SequentialDataStore/SDS_Streams.md [hunk 15]
```diff
@@ -109,64 +101,51 @@ SdsStream information.
...
-         "TypeId":"Simple",
-      }
+Example response body:
+```json
+HTTP/1.1 200
+Accept: application/json
 
+{  
+   "Id":"Simple"
+   "Name":"Simple"
```
[the remark this answers is not in the document]
Comment (mckenzielee) [reply Documentation/SequentialDataStore/SDS_Streams.md]: That helps identify issues with bad formatting. A comma is missing after "Id":"Simple"

**Pull request merged**